x/evm/types/predeploys.go

```diff
@@ -102,4 +103,15 @@ var (
 	}
+	// create3 - both versions of it:
+	// (1) deployed by the author using a known deployer address + nonce
+	// (2) deployed using create2, which differs by the metadata only
+	create3 = Predeploy{
+		Address: "0x9fBB3DF7C40Da2e5A0dE984fFE2CCB7C47cd0ABf",
```

Comment:
how is the contract address determined for predeployed contracts?

Reply:
Added a link to the repo just above. The address is listed in the README and, for this one, in an issue.